docs/getting-started/users-page.asciidoc

```diff
@@ -85,7 +85,12 @@ The risk summary visualization shows the user risk score and user risk level. Ho
 
 The risk summary table shows the category, score, and number of risk inputs that determine the user risk score. Hover over the table to display the **Inspect** button, which allows you to inspect the table's queries.
 
-To expand the **User risk summary** section, click **View risk contributions**. The left panel displays additional details about the user's risk inputs.
+To expand the **User risk summary** section, click **View risk contributions**. The left panel displays additional details about the user's risk inputs:
+
+* The asset criticality level and contribution score from the latest risk scoring calculation.
+* The top 10 alerts that contributed to the latest risk scoring calculation, and each alert's contribution score.
```

Comment:
```suggestion
* The top 10 alerts that contributed to the latest risk score, and each alert's contribution score.
```

Reply:
I feel that "risk scoring calculation" is a bit more precise here, so I'd prefer to leave it as is.